index.js

```diff
@@ -3104,7 +3104,7 @@
   def('trim',
       {},
       [$.String, $.String],
-      R.trim);
+      function(s) { return s.trim(); });
```

Comment:
> In a year, say, the number of people whose browsers don't provide String#trim will be lower than it is today, presumably.

That's probably true, but I think the number of organisations / apps that want to support ES3-only browsers changes much less gradually and more unpredictably